### PR TITLE
Enable `--force` flag in `flytectl demo start` and `flytectl demo start --dev`

### PR DESCRIPTION
**Pull request by Future-Outlier**
# TL;DR
Now, the user can use `flytectl demo start` and `flytectl demo start --dev` without passing `y`.
## Type
- [ ] Bug Fix
- [x] Feature
- [ ] Plugin

## Are all requirements met?

- [x] Code completed
- [ ] Smoke tested
- [x] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description
I've tried to use only 1 default config, but due to the circular import, I choose to pass the variable from the terminal to the sandbox config and then from the sandbox config to the docker config.
If there's any alternative solutions, feel free to ping me and I will make it better!

## Screenshot
image

### CODE REVIEW

**Comment by Future-Outlier** [comment]
@eapolinario Please take a look!
Thanks a lot!

**Review by pingsutw — APPROVED** [review]

**Review by eapolinario — APPROVED** [review]
